src/main/java/com/splunk/kafka/connect/SplunkSinkTask.java

```diff
@@ -238,6 +241,7 @@ public String headerId(SinkRecord sinkRecord) {
         if(sourcetypeHeader != null) {
             headerString.append(sourcetypeHeader.value().toString());
         } else {
+            if(metas!=null)
```

Comment:
The if can be formatted better with braces

Reply:
Added braces